Simulator/components/App.jsx

```diff
@@ -92,6 +97,16 @@ class App extends Component {
     const hasData = !!this.props.tables && Object.keys(this.props.tables).length > 0;
     const isAddressBarEmpty = (this.props.addressBarUrll === '');
 
+    const hasStandardConnections = this.props.standardConnections.length > 0;
+    if (hasStandardConnections) {
+      const connectionList = this.props.standardConnections;
+      standardConnectionsWindow = connectionList.map((standardConnection, idx) =>
```

Comment:
Its a list of tabs that each have a standard connection window inside, naming it to something like standardConnectionsList / TabList would make more sense though.